src/environmentd/tests/auth.rs

```diff
@@ -810,8 +810,8 @@ fn test_auth_expiry() {
     let encoding_key =
         EncodingKey::from_rsa_pem(&ca.pkey.private_key_to_pem_pkcs8().unwrap()).unwrap();
 
-    const EXPIRES_IN_SECS: u64 = 40;
-    const REFRESH_BEFORE_SECS: u64 = 20;
+    const EXPIRES_IN_SECS: u64 = 2;
```

Comment:
These were increased three separate times because they caused flakes:

https://github.com/MaterializeInc/materialize/commit/b0d262b04b859c2b94a6353c9172d966972b778f
https://github.com/MaterializeInc/materialize/commit/c1aae7bbcbd79972ea166dc628fa4cd167d343a0
https://github.com/MaterializeInc/materialize/commit/5f668064e9a7388711c6ef35ddd9bb0917482264

I'm not opposed to getting them down to 2 seconds, but we should only do this if we are certain they won't flake. It is possible there are deterministic or other safe ways to do this quickly in realtime that don't have flakes. Fake clock? Some improved retry or other algorithms? I don't care which, but would like to not reintroduce flakes. I'm also not convinced that bumping the retries like I did earlier was correct. It just made the flakes go away. Understanding a root cause would be better, but takes time.

Reply:
Oy, I did not realize they were so high to avoid test flakes. I'm just going to drop that commit and keep the one that breaks the tests up. We can open up a separate issue to look at that.

Reply:
https://github.com/MaterializeInc/materialize/issues/22145